components/createTaskModal/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { useMutation, useQuery } from '@apollo/client';
+import { useState } from 'react';
+import { Button, Form, Modal } from 'react-bootstrap';
+import allUsersQuery from './allUsers';
+import createTaskMutation from './createTask';
+
+interface Props {
+  showModal: boolean;
+  handleClose: () => void;
+  boardCategory: string;
+}
+
+const createTaskModal: React.FC<Props> = ({
+  showModal,
+  handleClose,
+  boardCategory,
+}) => {
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [assignTo, setAssignTo] = useState('');
+
+  const { data: usersData } = useQuery(allUsersQuery);
+  const [createTask, {}] = useMutation(createTaskMutation, {
+    onCompleted: () => {
+      setTitle('');
+      setDescription('');
+      setAssignTo('');
+    },
+  });
+
+  const onSubmit = (e) => {
```

Comment:
When I create new task in order to see it on the board I have to reload the page. It means that after creating it client doesn't fetch updated list. Is it ok for now?

Reply:
I am working on caching it on apollo client at front end or storing it in a local state